bigbluebutton-html5/imports/ui/components/whiteboard/component.jsx

```
@@ -500,92 +501,185 @@ const Whiteboard = React.memo((props) => {
       : calcedZoom;
   };
 
-  let cameraUpdateTimeoutId = null;
-
   const adjustCameraOnMount = (includeViewerLogic = true) => {
-    // Clear any existing timeout to prevent overlaps
-    if (cameraUpdateTimeoutId) {
-      clearTimeout(cameraUpdateTimeoutId);
-    }
+    const throwIfInvalid = (value, description) => {
+      if (!isFinite(value)) {
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)